{{cookiecutter.repo_name}}/pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+requires = [
+  "setuptools >= 40.9.0",
+  "versioneer-518",
+  "wheel",
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "{{cookiecutter.repo_name}}"
+version = "0.0.0"
+description = "{{cookiecutter.description}}"
+license = {{ "{" }}file = "LICENSE" {{ "}" }}
+authors = [
+    {{ "{" }}name = "{{cookiecutter.author_name}}", email = "{{cookiecutter.author_email}}"{{ "}" }},
```

Comment:
The fact that the examples offer both `{}` on the end and `{}` for each entry just confuses me... https://peps.python.org/pep-0621/#example

Reply:
have a look at how it's rendered in the example repos in my account. the extra `{` and stuff are to get around cookiecutter/jinja templating. The syntax should be 621 compliant.

Reply:
Yeah sorry this wasn't a comment that the way this is doing it is wrong, just that I hate that the two equivalent entries are formatted differently. Indeed I think you could make this a list of list and the whole thing would still work...